README.md

```diff
@@ -137,12 +138,27 @@ Possible values are:
 * Sensor
 
 A sensor section could thus be named `Sensor_Heartbeat` or `Actuator1`.
+Section names have to be unique.
+
+## Default section
+
+Optionally a `DEFAULT` section can be added to the configuration.
+Parameters within this section will be the default for all sensors and actuator.
+Sensors and actuators can override the default if they specifie the same parameter.
+This is useful when many sensors of the same type with similar parameters are used.
+E. g. set the default for `TempUnit` so the DhtSensors don't need to specifie it repetitive:
+
+```yaml
+DEFAULT:
+    TempUnit: F
+```
 
 # Release Notes
 This current version is a nearly complete rewrite of the previous version with a number of breaking changes.
 
 ## Breaking Changes
 
+- The configuration file is now in YAML syntax insted of a ini file - March 2022
```

Comment:
Because of me we need to change that date.

Reply:
sure